conans/client/graph/build_mode.py

```diff
@@ -9,7 +9,7 @@ class BuildMode:
                    => ["hello/0.1@foo/bar"] if user wrote "--build hello/0.1@foo/bar"
                    => False if user wrote "never"
                    => True if user wrote "missing"
-                   => ["!foo"] means exclude when building all from sources
+                   => ["!foo"] or ["~foo"] means exclude when building all from sources
```

Comment:
Same, remove legacy from docstrings

Reply:
If this is just code documentation (not displayed in the help commands), I'd prefer to keep it consistent with the code

Reply:
I agree with Dani here 👍

Reply:
Yes, sorry, I thought it was part of the things that get automatically published in docs from docstrings. Of course, as long as it doesn't appear in the user docs or in ``conan -h``, it is better to leave it.